pkg/bgpv1/agent/controller.go

```diff
@@ -217,7 +172,12 @@ func (c *Controller) Run(ctx context.Context) {
 			"component": "Controller.Run",
 		})
 	)
-	l.Debug("Starting informers")
+	l.Info("Starting LocalNodeStore Observer")
+
+	// setup a reconciliation trigger on LocalNodeStore changes
+	c.LocalNodeStore.Observe(ctx, func(node node.LocalNode) { c.Sig.Event(struct{}{}) }, func(err error) {
```

Comment:
General observation, observe is not triggering event on changes to annotations in node object.

Reply:
You're right, issue is here: https://github.com/ldelossa/cilium/blob/ldelossa/bgpv1-configreconciler-hivecell/pkg/endpointmanager/host.go#L70

We do not trigger any update to the LocalNodeStore on k8s node updates.